integration_tests/models/schema_tests/schema.yml

```diff
@@ -166,6 +166,8 @@ models:
             column_list: ["idx", "date_col", "col_numeric_a", "col_numeric_b", "col_string_a", "col_string_b", "col_null"]
         - dbt_expectations.expect_table_column_count_to_equal_other_table:
             compare_model: ref("data_test")
+        - dbt_expectations.expect_table_columns_to_not_contain_set:
+            column_list: ["col_numeric_b", "col_string_a"]
```

Comment:
this should probably be something like
```
column_list: ["col_numeric_c", "col_string_d"]
```
to make the test pass since we're looking for columns *not* in the model?

Reply:
Updated and tests passed